### ISSUE TITLE
Update to Boostrap 5

### ISSUE THREAD

**Issue by hupf** [swiss/styleguide#661]
Is this project still maintained? It depends on Bootstrap 3.3.7 while the latest v3 release is 3.4.1. Also, starting a new project with Bootstrap 3, while Bootstrap 5 is available, feels a bit funky.

As far as I understand, the only alternative is https://oblique.bit.admin.ch/ – which is tied to Angular. It would be nice to have a up-to-date, framework-agnostic implementation of the Swiss Confederation Web Guidelines (see also: https://github.com/oblique-bit/oblique/issues/39).

**Comment by Soardiac** [comment]
There is a renewal project under way. Refreshed design, different technology. It isn't published yet because everything is still changing. **But this bootstrap based styleguide will be discontinued.**
There will be a design system, based on the [tailwind CSS framework](https://tailwindcss.com/), published on [storybook](https://storybook.js.org/).

**Comment by hupf** [comment]
Thanks @Soardiac for the news, this sounds very promising!

Is there any rough E.T.A. when the new project will be released?

**Comment by Soardiac** [comment]
I don't know when a first version will be published. But I hope it will be within weeks.

**Comment by msio** [comment]
I'm also interested in new version. thanks for any updates

**Comment by msio** [comment]
Will be also possible to use the library without bootstrap at all ?

**Comment by gillerr** [comment]
@msio , yes, the new version won't use Boostrap at all.

**Comment by msio** [comment]
@gillerr thanks that's great news. anyway any update on release date :D ?

**Comment by hupf** [comment]
I'd like to share a few informations I found out upon further inquiries, that may make it easier for others to understand the state of this project and its rewrite:

- The latest version of the current Swiss Styleguide (3.0.0) is actually 6 years old and containsm, according to `npm audit`, 304 security vulnerabilities in its dependencies. Due to the re-implementation, I doubt that there will be any maintenance work fixing this situation.
- The Tailwind CSS rewrite of it, is not only a technical rewrite, but also implements a new corporate design. All of this is going to be officially launched on 01.01.2023
- Any sites (or «Fachanwendungen») that want to launch with this new version before the above date – or before the main department's website switched to the new design – must apply to certain conditions and have an official approval by a central goverment entity, to be able to do so.
- The new implementation will, as far as I understand, be more like a set of Tailwind-styled markup to be copy/pasted in your project and won't contain any interaction. It is expected that the projects themselves will implement all the logic in their appropriate JavaScript framework or library (like dropdown interaction, keyboard navigation etc.). I hope they will at least provide a reference implementation in vanilla JavaScript (or even Web Components that are available in all mainstream browsers by now) – but it didn't sound like they will.

So to summarize, the situation for sites planning to launch in the next few months is a bit dodgy.

For more information you may contact webforum@bk.admin.ch.

**Comment by darioschwab** [comment]
Two years in, I would expect this rehaul to have moved on, but I am unable to find any information. This thread mentions 01.01.2023 as a release date. Looking at the current version of the READ.me file, this seems to have been moved back to July 2023. Both of these dates are long gone. Preparing for another website project for a federal office, we really would like to avoid building on a library that is eight years old. Where can we get our hands on the rewrite, be it only a preliminary version?
